README.md

```diff
@@ -1,2 +1,55 @@
 # terraform-aws-databricks
```

Comment:
Please add a description/write-up or at least some info - even if you're linking to another readme. This isn't a great first-read for people landing at the repo.